.github/workflows/21.04.yml

```diff
@@ -0,0 +1,153 @@
+name: Unix
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+  release:
+    types: # This configuration does not affect the page_build event above
+      - created
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  main:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        cubit: [2021.5]
+        os: [ubuntu-20.04]
```

Comment:
```suggestion
        os: [ubuntu-21.04]
```

Reply:
Just going to commit this suggestion to retrigger the CI and generate another assert for testing locally

Reply:
we need 20.04 GitHub action doesn't support 21.04 just yet...
this is running in a docker...